src/game_api/script/lua_vm.cpp

```diff
@@ -990,8 +993,8 @@ std::shared_ptr<sol::state> acquire_lua_vm(class SoundManager* sound_manager)
     {
         assert(sound_manager != nullptr && "SoundManager needs to be passed to first call to get_lua_vm...");
 
-        std::shared_ptr<sol::state> global_vm = std::make_shared<sol::state>();
-        sol::state& lua_vm = *global_vm;
+        std::shared_ptr<sol::state> global_vms = std::make_shared<sol::state>();
```

Comment:
What is `vms` supposed to me?

Reply:
just changed the name cuz of static above